site/en/workflows.md

```diff
@@ -1602,7 +1602,7 @@ results by configuring a more granular workflow like e.g. the
 ### Example with ocrd-process
 
 ```sh
-ocrd process "tesserocr-recognize -P segmentation_level region -P textequiv_level word -P find_tables true -P model GT4HistOCR_50000000.997_191951"
+ocrd process "tesserocr-recognize -P segmentation_level region -P textequiv_level word -P find_tables true -P model GT4HistOCR_50000000.997_191951 -I OCR-D-IMG -O OUTPUT"
```

Comment:
```suggestion
ocrd process "tesserocr-recognize -P segmentation_level region -P find_tables true -P find_staves false -P model Fraktur_GT4HistOCR -I OCR-D-IMG -O OCR-D-OCR"
```
changes explained:
- `textequiv_level=word` is the default, 
- so is `staves=false` and `tables=true`, but it might be relevant to know that even for the casual user
- the `GT4HistOCR_50000000.997_191951` model is not registered anywhere – but this one is
- output fileGrp should abide by our conventions

You might also want to add:
```sh
ocrd resmgr download ocrd-tesserocr-recognize Fraktur_GT4HistOCR.traineddata
```

Reply:
Same thing goes for the Calamari `qurator-gt4histocr-1.0` model BTW.

And the language regarding its `checkpoint` parameter must also be rewritten – we just changed that to `checkpoint_dir` only.

Reply:
Yes I noticed the model issue later, too.
About the default params, maybe they can be omitted completely then. People who want to execute that command may just want to see quick results.

Reply:
> About the default params, maybe they can be omitted completely then. People who want to execute that command may just want to see quick results.

I think these workflow examples are more than just "take it or leave it". They also illustrate the general mechanism (processor steps and parameter choices) and can be easily customized. So informing the users (even the unpatient ones) about relevant possibilities along with the preconfigured choices is worthwhile. (We already follow that pattern in the step examples above.)

Reply:
@paulpestov To me, this was not resolved. The current recommendations are **not** copy-pastable (model download, `checkpoint` param of Calamari) – see above.

Reply:
After the changes the command became copy-pastable for me so it fixed the issue this PR was about. Maybe we should handle the remaining issues in a separate PR.